using-mobile-extensions/adobe-audience-manager/README.md

```diff
@@ -22,15 +22,37 @@ To get started with **Audience Manager**, follow these steps:
 5. Click **Save**.
 6. Follow the publishing process to update SDK configuration.
 
-## Add Audience Manager to your App
-
-**Java**
+### Add Audience Manager to your App
 
+{% tabs %}
+{% tab title="Android" %}
 1. Add the library to your project.
 2. Import the library:
 
+#### Java
 `import com.adobe.marketing.mobile.*;`
 
+{% tab title="iOS" %}
+1. Add the library to your project via your `Podfile` by adding `pod 'ACPAudience'`
+2. Import the Audience and Identity library:
+
+#### Objective-C
+  ```objectivec
+  #import "ACPCore.h"
+  #import "ACPAudience.h"
+  #import "ACPIdentity.h"
+  ```
+
+#### Swift
+  ```swift
+   import "ACPCore"
```

Comment:
These should be just without the quotes
```
import ACPCore
import ACPAudience
```

When installing via Cocoapods the module map only creates ACPCore which has an umbrella header that auto imports ACPIdentity. https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/31